dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/AbstractViewController.java

```diff
@@ -49,7 +49,11 @@ private void onMousePressed(MouseEvent e) {
 
 	@SuppressWarnings("unused") @FXML
 	private void onScroll(ScrollEvent e) {
-		view.zoom(e.getDeltaY(), new Point2D(e.getX(), e.getY()));
+		if (e.getDeltaY() > 0) {
+			view.zoomInScroll(e.getX(), e.getY());
+		} else if (e.getDeltaY() < 0) {
```

Comment:
Can the y delta actually be zero? Seems illogical to me. If so, this can just be `else`